### PR TITLE
Progress page data fetching

### PR DESCRIPTION
**Pull request by efischer19**
@nasthagiri @jcdyer 

The tests in these files were already set up to do a "realistic" load test of all courseware, so I just removed everything that wasn't the progress page. Bam, a load test for the progress page.

### CODE REVIEW

**Comment by jcdyer** [comment]
Magic.

**Comment by nasthagiri** [comment]
Wait, so you're not actually merging this, right?  Just using this branch to test the progress page?

**Comment by nasthagiri** [comment]
Against which course (with how many blocks in the course) will this be run?

**Comment by efischer19** [comment]
I was thinking MReVx is decently large: https://courses-loadtest.edx.org/courses/MITx/8.MReVx/2T2014/progress

And no, I won't be merging this - I'll just run the tests off my machine and close the PR once we're done.

**Comment by jcdyer** [comment]
Is there a way to turn off the big green merge button?

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ install:
     - pip install --upgrade pip wheel pep8
 script:
     - pep8 --config=.pep8 .
+    - exit 1
```
Comment (efischer19) [inline .travis.yml]: @jcdyer This kills the button in travis CI, must-pass-checks-before-merging repos

Comment (jcdyer) [inline .travis.yml]: Spiffy.

**Comment by efischer19** [comment]
@jcdyer Just playing with the new "multiple assignees" feature. Did you get a notification for it?

I wonder if we ought to consider using this functionality to tag reviewers - it would make https://github.com/pulls/assigned pretty useful for finding "PRs I need to take action on"

**Comment by efischer19** [comment]
Summary of my results can be seen here: https://openedx.atlassian.net/wiki/display/TNL/Progress+Page+Load+Testing+Results

Closing this PR as no work remains to be done/merged here.

**Pull request closed without merging**